style.css

```diff
@@ -1,42 +1,54 @@
+
 body {
   font-family: Poppins, Arial, Helvetica, sans-serif;
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
-  margin-left: 24px;
-  margin-right: 24px;
 }
 
 nav {
   display: flex;
   justify-content: space-between;
   align-items: center;
+  margin-left: 24px;
+  margin-right: 24px;
 }
 
-.headline {
+section {
   background-image: url(/images/header-shapes\ mobile@2x.svg);
```

Comment:
**[REQUIRED BUT OPTIONAL]**

# FIX BEFORE YOU MERGE

- Great job implementing the changes, however, here's something to bear in mind. Your image ain't showing now as a result of how you've named it in the images directory. it would show locally but won't on GitHub because GitHub is case-sensitive. You can fix this by removing those characters in its name and correcting the file path like this below:

`header-shapes-mobile.svg`
`background-image: url(./images/header-shapes-mobile.svg);`
- As you've also used the section tag to add styling, do bear in mind that this project would have many sections as you continue building it, so you should give it a CSS class name to avoid all your sections having the same background later. As this issue might not be obvious to you now but later.
- Also, let's add some margin-top, like 40px or 54px to move the section down a little from the header to improve our styling.